### PR TITLE
adding some glossaries for docs-reference-glossary

### PR DESCRIPTION
**Pull request by nguyenuet**
Add some glossaries: cluster, containerd, control-plane, cri-o, daemonset, etcd, kube-apiserver, kube-scheduler

### CODE REVIEW

File: content/vi/docs/reference/glossary/cri-o.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: CRI-O
+id: cri-o
+date: 2020-03-05
+full_link: https://cri-o.io/#what-is-cri-o
+short_description: >
+  A lightweight container runtime specifically for Kubernetes
+  Một lightweight container runtime đặc biệt cho Kubernetes
```
Comment (sftim) [inline content/vi/docs/reference/glossary/cri-o.md]: The English is still here.

File: content/vi/docs/reference/glossary/cri-o.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<!--more-->
+
+CRI-O is an implementation of the {{< glossary_tooltip term_id="cri" >}}
+to enable using {{< glossary_tooltip text="container" term_id="container" >}}
+runtimes that are compatible with the Open Container Initiative (OCI)
+[runtime spec](http://www.github.com/opencontainers/runtime-spec).
+
+Deploying CRI-O allows Kubernetes to use any OCI-compliant runtime as the container
+runtime for running {{< glossary_tooltip text="Pods" term_id="pod" >}}, and to fetch
+OCI container images from remote registries.
```
Comment (sftim) [inline content/vi/docs/reference/glossary/cri-o.md]: The English is still here.

File: content/vi/docs/reference/glossary/cri-o.md (hunk 1 above)
Comment (sftim) [reply content/vi/docs/reference/glossary/cri-o.md]: I would expect to see “lightweight” translated.

Comment (nguyenuet) [reply content/vi/docs/reference/glossary/cri-o.md]: thank you! I will update it

File: content/vi/docs/reference/glossary/cri-o.md (hunk 2 above)
Comment (nguyenuet) [reply content/vi/docs/reference/glossary/cri-o.md]: Thank you. I am sorry for this mistake. I will update

File: content/vi/docs/reference/glossary/cluster.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Cluster
+id: cluster
+date: 2020-02-26
+full_link: 
+short_description: >
+   Một tập các worker machine, được gọi là node, dùng để chạy các containerized application. Mỗi cụm (cluster) có ít nhất một worker node.
```
Comment (truongnh1992) [inline content/vi/docs/reference/glossary/cluster.md]: `containerized application` should be `các ứng dụng được đóng gói`

File: content/vi/docs/reference/glossary/cluster.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+aka: 
+tags:
+- fundamental
+- operation
+---
+Một tập các worker machine, được gọi là node, dùng để chạy các containerized application. Mỗi cụm (cluster) có ít nhất một worker node.
+
+<!--more-->
+Các worker node chứa các pod (là những thành phần của ứng dụng). Control Plane quản lý các worker node và pod trong cluster.
+Trong môi trường sản phẩm (production environment), Control Plane thường chạy trên nhiều máy tính và một cluster thường nhạy trên nhiều nodes, cung cấp khả năng chịu lỗi (fault-tolerance) và tính sẵn sàng cao (high availability).
```
Comment (truongnh1992) [inline content/vi/docs/reference/glossary/cluster.md]: `nhạy` -> `chạy`

File: content/vi/docs/reference/glossary/kube-apiserver.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+- architecture
+- fundamental
+---
+ API server là một thành phần của Kubernetes {{< glossary_tooltip text="control plane" term_id="control-plane" >}}, được dùng để đưa ra Kubernetes API.
+API server là front end của Kubernetes control plane.
+
+<!--more-->
+
+Thực thi chính của API server là [kube-apiserver](/docs/reference/generated/kube-apiserver/).
+kube-apiserver được thiết kế để co giãn theo chiều ngang &mdash; có nghĩa là nó co giãn bằng cách triển khai thêm các thực thể.
```
Comment (truongnh1992) [inline content/vi/docs/reference/glossary/kube-apiserver.md]: what is `@mdash`?

File: content/vi/docs/reference/glossary/cri-o.md (hunk 2 above)
Comment (truongnh1992) [reply content/vi/docs/reference/glossary/cri-o.md]: @sftim these words are primitive and no need to translate to Vietnamese. Thank you for your review.

**Comment by truongnh1992** [comment]
@nguyenuet welcome you to K8s Vietnamese localizing team :D
Thank you for your PR.

File: content/vi/docs/reference/glossary/kube-apiserver.md (hunk 5 above)
Comment (nguyenuet) [reply content/vi/docs/reference/glossary/kube-apiserver.md]: This is "—" . For example in this case, it is "co giãn theo chiều ngang — có nghĩa là"

**Comment by truongnh1992** [comment]
/lgtm
/approve
Thank @nguyenuet

**Pull request merged**